electrum/commands.py

```diff
@@ -1322,6 +1322,34 @@ async def reverse_swap(self, lightning_amount, onchain_amount, wallet: Abstract_
             'onchain_amount': format_satoshis(onchain_amount_sat),
         }
 
+    @command('n')
+    async def convert_currency(self, from_amount, from_ccy = None, to_ccy = 'BTC'):
+        """Converts the given amount of currency to another using the
+        configured exchange rate source.
+        """
+        # Currency codes are uppercase
+        from_ccy = self.daemon.fx.ccy if from_ccy is None else from_ccy.upper()
+        to_ccy = to_ccy.upper()
+        # Get current rates
+        rate_from = self.daemon.fx.exchange.get_cached_spot_quote(from_ccy)
+        rate_to = self.daemon.fx.exchange.get_cached_spot_quote(to_ccy)
```

Comment:
Note that which ccy rates are available depends on the configured exchange (config key `use_exchange`) and the configured currency (config key `currency`). In particular for quite a few exchanges, only the `fx.ccy`-`BTC` fx rate is available (depends on the `ExchangeBase.get_rates` implementation).
This should be noted in the docstring.

-----

In particular note that the "BTC" to "BTC" exchange rate is not guaranteed to be present.
```suggestion
        rate_from = self.daemon.fx.exchange.get_cached_spot_quote(from_ccy) if from_ccy != "BTC" else Decimal(1)
        rate_to = self.daemon.fx.exchange.get_cached_spot_quote(to_ccy) if to_ccy != "BTC" else Decimal(1)
```
Alternatively, you can modify `exchange.get_cached_spot_quote()` to special-case "BTC" - that might be cleaner actually.

Reply:
I decided to modify `exchange.get_cached_spot_quote()` as you suggested, I'll add that commit to this PR.